File: 02_devApplication_IOS_avec_Objective_C/10_UITableVieux_CellulePerso_MasterDetail/10_UITableVieux_CellulePerso_MasterDetail/DetailViewController.m
//
//  DetailViewController.m
//  10_UITableVieux_CellulePerso_MasterDetail
//
//  Created by etudiant on 4/24/17.
//  Copyright © 2017 etudiant. All rights reserved.
//

#import "DetailViewController.h"

@interface DetailViewController ()
@property (weak, nonatomic) IBOutlet UIImageView *imgGateau;
@property (weak, nonatomic) IBOutlet UILabel *etqTitre;

@property (weak, nonatomic) IBOutlet UILabel *etqDescription;


@end

@implementation DetailViewController

- (void)viewDidLoad {
    [super viewDidLoad];
  
    self.etqTitre.text = self.gateau.nom;
    self.etqDescription.text = self.gateau.texte;
    self.imgGateau.image = [UIImage imageNamed:self.gateau.nomImage];
    
}

- (IBAction)btnRetourTached:(id)sender
{
    [self dismissViewControllerAnimated:YES completion:nil];
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: 04_devApplication_IOS_avec_Swift/07_TableView/07_TableView/DetailViewController.swift
//
//  DetailViewController.swift
//  07_TableView
//
//  Created by etudiant on 5/12/17.
//  Copyright © 2017 etudiant. All rights reserved.
//

import UIKit

class DetailViewController: UIViewController {

    @IBOutlet weak var etqImage: UIImageView!
    
    @IBOutlet weak var etqTitre: UILabel!
    @IBOutlet weak var etqDescription: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func backButtonPush(_ sender: UIBarButtonItem) {
        dismiss(animated: true, completion: nil)
    }

   

}
